test/shared_setup.py

```diff
@@ -127,29 +127,30 @@ def test_action_target(self):
         """Test action_target function, mostly w.r.t. backward compatibility."""
         def fake_setup(*args, **kwargs):
             """Fake setup function to test action_target with."""
-            print 'args: ', args
-            print 'kwargs:', kwargs
+            print('args: %s' %''.join(args))
+            print('kwargs: %s' %kwargs)
 
         self.mock_stdout(True)
         action_target({'name': 'vsc-test', 'version': '1.0.0'}, setupfn=fake_setup)
         txt = self.get_stdout()
         self.mock_stdout(False)
-
-        self.assertTrue(re.search(r"^args:\s*\(\)", txt, re.M))
-        self.assertTrue(re.search(r"^kwargs:\s*\{.*'name':\s*'vsc-test'", txt, re.M))
+        self.assertTrue(re.search(r"args:.*\(\)", txt, re.M))
```

Comment:
@vsoch You can put back the `^` after fixing the `print` statements in `fake_setup` above?

These checks shouldn't need any tweaks imho...

Reply:
I think they would, they return different orderings of the keys?

Reply:
These tests were originally failing because the `print` statements `fake_setup` produced different output when using Python 3.x, but now they shouldn't anymore because you changed them...

The original checks should now (still) pass, even with `^args:\s*` and `^kwargs:\s*`?

Reply:
see https://github.com/vsoch/vsc-install/pull/1 + https://travis-ci.org/boegel/vsc-install/builds/378177768